inference-engine/tests/functional/plugin/shared/CMakeLists.txt

```diff
@@ -27,6 +27,10 @@ addIeTarget(
             ${EXPORT_DEPENDENCIES}
 )
 
+if (TARGET MKLDNNPlugin)
+    add_dependencies(${TARGET_NAME} MKLDNNPlugin)
+endif()
```

Comment:
Is it okay to add as dependency MKLDNNPlugin only, while PR name is about multi and heter as well?